apps/teams-test-app/src/components/DialogAPIs.tsx

```diff
@@ -68,21 +69,41 @@ const DialogAPIs = (): ReactElement => {
       },
     });
 
+  const UpdateTaskModule = (): ReactElement =>
+    ApiWithTextInput<TaskInfo>({
+      name: 'updateTaskModule',
+      title: 'Update Task Module',
+      onClick: {
```

Comment:
This seems unnecessary since you accomplish the same goal with the `withCallback` call

Reply:
Deleted. I think it is a little confusing that we are using 'withPromise' for v2 even though, we actually are using callbacks.